inclusive-design/wecount.inclusivedesign.ca
# Split news and views

## Description
To address https://github.com/inclusive-design/wecount.inclusivedesign.ca/issues/89. It includes:

1. Split the "News and Views" page into two separate pages: "News", "Views";
2. Clicking on a news item redirects the user to the external news;
3. Clicking on a views item brings up an internal page showing its content;
4. Refactored the store to fetch data only once when the website starts;
5. Add "numOfRecPerPage" in config.js to control the number of records shown for one page. This value applies to news, items and search pages.

## Steps to test

1. Run `npm run dev` to start the website;
2. Open the site at http://localhost:3000;
3. The feature described above should match
